Makefile

```diff
@@ -3,7 +3,8 @@ GOARCH := $(shell go env GOARCH)
 .PHONY: setup # Set up required tools (builder, mdatagen)
 setup:
 	go install go.opentelemetry.io/collector/cmd/builder@latest
-	go install github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen@latest
+	chmod +x ./scripts/setup_mdatagen.sh
+	./scripts/setup_mdatagen.sh
```

Comment:
Why that change? `go install` seems to be working fine, we can also add some info to ensure that GOPATH is added to PATH variable:

```
@echo "\nNOTE: Ensure that" `go env GOPATH` "/bin is in your PATH env variable"
```

Reply:
removed script, will be proceeding `opentelemetry-collector-contrib` way